integrations/pull_merge_test.go

```diff
@@ -77,6 +77,20 @@ func TestPullRebase(t *testing.T) {
 	testPullMerge(t, session, elem[1], elem[2], elem[4], models.MergeStyleRebase)
 }
 
+func TestPullRebaseMerge(t *testing.T) {
+	// TODO
```

Comment:
Initially I inteded to modify the test case so that an actual rebase is performed and necessary.

The test `TestPullRebase` does not create such a setting, so i just removed the comments.